src/views/settings/SettingsMultiServiceSpace.tsx

```diff
@@ -0,0 +1,427 @@
+import {useMultiService} from "@/stores/multiService";
+import {MultiServiceFeature} from "@/stores/multiService/types";
+import LottieView from "lottie-react-native";
+import {anim2Papillon} from "@/utils/ui/animations";
+import Reanimated, {FadeOut, ZoomIn} from "react-native-reanimated";
+import PapillonBottomSheet from "@/components/Modals/PapillonBottomSheet";
+import * as Haptics from "expo-haptics";
+import AccountItem from "@/components/Global/AccountItem";
+
+const SettingsMultiServiceSpace: Screen<"SettingsMultiServiceSpace"> = ({ navigation, route }) => {
+  const theme = useTheme();
+  const insets = useSafeAreaInsets();
+  const space = route.params.space;
+  const accounts = useAccounts();
+  const availableAccounts = accounts.accounts.filter(account => !account.isExternal && !(account.service == AccountService.PapillonMultiService));
+  const deleteMultiServiceSpace = useMultiService(store => store.remove);
+  const updateMultiServiceSpace = useMultiService(store => store.update);
+  const setMultiServiceSpaceAccountFeature = useMultiService(store => store.setFeatureAccount);
+
+  const linkedAccount = accounts.accounts.find(account => account.localID === space.accountLocalID) as PrimaryAccount | undefined;
+
+  const firstNameRef = useRef<TextInput>(null);
+  const lastNameRef = useRef<TextInput>(null);
+  const spaceNameRef = useRef<TextInput>(null);
+
+  const [firstName, setFirstName] = useState(linkedAccount?.studentName?.first ?? "");
+  const [lastName, setLastName] = useState(linkedAccount?.studentName?.last ?? "");
+  const [spaceName, setSpaceName] = useState(space.name);
+
+  const [loadingImage, setLoadingImage] = useState(false);
+  const [selectedImage, setSelectedImage] = useState<null | string>(null);
+  const selectPicture = async () => {
+    setLoadingImage(true);
+
+    let result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.Images,
+      allowsEditing: true,
+      aspect: [1, 1],
+      quality: 1,
+      base64: true,
+    });
+
+    if (!result.canceled) {
+      const img = "data:image/jpeg;base64," + result.assets[0].base64;
+      updateMultiServiceSpace(space.accountLocalID, "image", img);
+      // @ts-expect-error
+      accounts.update(space.accountLocalID, "personalization", {
+        profilePictureB64: img
+      });
+      setSelectedImage(img);
+    }
+
+    setLoadingImage(false);
+  };
+
+  const [accountSelectorOpened, setAccountSelectorOpened] = useState(false);
+  const [selectedAccount, setSelectedAccount] = useState<PrimaryAccount | null>(null);
+  const [featureSelection, setFeatureSelection] = useState<MultiServiceFeature>(MultiServiceFeature.Grades);
+
+  const deleteSpace = () => {
+    Alert.alert("Êtes-vous sur ?", "Cette action entrainera la suppression de votre espace multi-service.", [ { text: "Annuler", style: "cancel" }, { text: "Confirmer", style: "destructive", onPress: () => {
```

Comment:
Le manque de retour à la ligne 😭

Reply:
Faudra supprimer les 2 prochaines lignes

```suggestion
    Alert.alert(
      "Êtes-vous sur ?",
      "Cette action entrainera la suppression de votre espace multi-service.",
      [
        { text: "Annuler", style: "cancel" },
        {
          text: "Confirmer",
          style: "destructive",
          onPress: () => {
            accounts.remove(space.accountLocalID);
            deleteMultiServiceSpace(space.accountLocalID);
            navigation.goBack();
          },
        },
      ]
    );
```